build_tools/github_actions/configure_ci.py

```diff
@@ -132,6 +132,7 @@ def contains(cls, val):
 # The file paths should be specified using Unix shell-style wildcards.
 PRESUBMIT_TOUCH_ONLY_JOBS = [
     ("build_test_all_macos_arm64", ["runtime/src/iree/hal/drivers/metal/*"]),
+    ("build_test_all_windows", ["*win32*", "*windows*", "*msvc*"]),
```

Comment:
Looking for other paths that have broken due to not running presubmits:

https://github.com/openxla/iree/commit/8ab9e12b0bf7721a648710efbdb24db41f962c10: `compiler/bindings/python/iree/compiler/api/ctypes_dl.py`
https://github.com/openxla/iree/commit/b7c942a29c0dbb54a18280dbb0bb5697268c1879 / https://github.com/openxla/iree/commit/65817331c95a48ac3f7d972667f4bd2c35b8f61e: `runtime/bindings/python/iree/runtime/system_api.py` -> `tests/e2e/models/mnist_train_test/mnist_train_test.py`

Not sure how we'd include those paths/targets with simple rules.